### PR TITLE
Chore: start fixing sdk/python/dev

### PR DESCRIPTION
**Pull request by shykes**
- Remove deprecated 'root' field
- Convert filters on deprecated root, to a view
- Update broken dependency

Note: the module itself still doesn't build

Signed-off-by: Solomon Hykes <solomon@dagger.io>

### CODE REVIEW

File: sdk/python/dagger.json
Comment (jedevc) [inline sdk/python/dagger.json:1]: Why is this a separate file?

Comment (jedevc) [reply sdk/python/dagger.json:1]: Also, huh, do views allow you to split out onto separate files like this? If so, nice 🎉 I didn't know that

Comment (shykes) [reply sdk/python/dagger.json:1]: > Why is this a separate file?

- `sdk/python/dev/dagger.json`: an actual dagger module, to help develop the Python SDK
- `sdk/python/dagger.json`: configure views for `sdk/python/`, for consumption by other modules

**Comment by helderco** [comment]
Please don’t waste your time here.

I have a draft pr for this but it’s very outdated. I have more local changes that I haven’t submitted. It’s not just a fix for the removed root, I’m changing more things. That took me on a curve to explore some new tools in the ecosystem like rye, uv, and changes in hatch, for a std module, including exploring multiple ways to structure said dagger module. In the end I only moved forward implementing some of that in the runtime module but then had to prioritize other things like the CLI. 

Just wanted to say that this is not forgotten and there’s more changes coming for the Python dev module that will replace this and the Python part of the ci module.

**Comment by shykes** [comment]
Thanks for the info. Not a waste of time so far.

**Comment by shykes** [comment]
Since @helderco is already working on a python implementation that's further ahead than #6971, I'm going to focus on the interface between top-level module and each per-SDK module. I'll make my skeleton implementation in Go, to move faster, and emphasize that it's a throwaway, to be replaced by the more complete, and more native, Python implementation.

**Pull request closed without merging**